Dockerfile

```
@@ -0,0 +1,54 @@
+FROM ubuntu
```

Maybe use a node image instead?
```suggestion
FROM node:11
```